lib/3scale/backend/listener.rb

```diff
@@ -413,10 +415,14 @@ def do_api_method(method_name)
       ##
       ##
       post '/transactions.xml' do
+
```

Comment:
extra blank line here

Reply:
I have fixed the formatting issues with blank the blank lines in a new forced push